etcdctl/command/util.go

```diff
@@ -217,6 +222,9 @@ func mustNewClient(c *cli.Context) client.Client {
 	}
 
 	if !c.GlobalBool("no-sync") {
+		if c.GlobalBool("debug") {
+			fmt.Fprintf(os.Stderr, "fetching the whole set of advertised client urls as cluster endpoints\n")
```

Comment:
can we simply print out something like "start to sync cluster" and print out the result of sync?